### PR TITLE
Fix tzinfo today

### PR DESCRIPTION
**Pull request by stdweird**

### CODE REVIEW

**Comment by stdweird** [comment]
@wpoely86 @wdpypere review pls

File: lib/vsc/utils/timestamp.py [hunk 1]
```diff
@@ -44,7 +44,7 @@
 
 def convert_to_datetime(timestamp=None):
     """
-    Convert a string or datetime.datime instance to a datetime.datetime with local tzinfo
+    Convert a string or datetime.datime instance to a datetime.datetime with UTC tzinfo
```
Comment (wpoely86) [inline lib/vsc/utils/timestamp.py]: this will not break anything else?

Comment (stdweird) [reply lib/vsc/utils/timestamp.py]: it's a bug. if fixing the bug breaks other stuff, should fix the other bugs too

**Comment by stdweird** [comment]
@wpoely86 thanks

**Pull request merged**